.gitbook/FAQ.md

```diff
@@ -49,19 +51,19 @@ If your wallet does not support this feature, you will have to connect manually.
 
 To move assets into or out of an Optimistic Ethereum network you can use the [**Boba Gateway**](https://gateway.boba.network). Detailed instructions in our [user documentation](contents/user/001\_how-to-bridge.md).
 
-If you are a developer, you can choose any of these two methods: [the **classical bridge**, or the **light bridge**](contents/boba-basics/bridge-basics/).
+You can choose any of these two methods: [the **classical bridge**, or the **light bridge**](contents/boba-basics/bridge-basics/).
 
 ## Can I cancel a withdrawal after it has been submitted?
 
 _No_, withdrawals currently cannot be cancelled once submitted.
 
 ## Can I transfer directly from Boba to a centralized exchange?
 
-Only if the centralized exchange supports Boba Network (at this time none of them do). Hence, you have to bridge your assets into a network they do support, such as L1.
+Only if the centralized exchange supports Boba Network. Hence, you have to bridge your assets into a network they do support, such as L1.
```

Comment:
```suggestion
Only if the centralized exchange supports Boba Network. For exchanges that do not directly support Boba Network, you can still bridge your assets into a network they do support (such as Eth Mainnet or BNB Chain).
```

Doesn't need to be this text exactly, but the 'Hence', doesn't read correctly to me.